src/features/unified-share-modal/__tests__/SharedLinkPermissionMenu.test.js

```diff
@@ -84,6 +84,45 @@ describe('features/unified-share-modal/SharedLinkPermissionMenu', () => {
 
             expect(wrapper.find('LabelPillText')).toHaveLength(length);
         });
+
+        test.each`
+            isSharedLinkEditTooltipShown | length | should
+            ${true}                      | ${1}   | ${'should render LabelPillText if tooltip was shown'}
```

Comment:
We generally avoid parameterizing the human-readable test description. Should these be two separate tests?

Reply:
Ah ok. Yeah I'll split these up. Will definitely be more readable that way